src/DynamoApplications/StartupUtils.cs

```diff
@@ -13,19 +13,31 @@
 using Dynamo.Updates;
 using DynamoApplications.Properties;
 using DynamoShapeManager;
+using DynamoUtilities;
 using Microsoft.Win32;
 using NDesk.Options;
 
 namespace Dynamo.Applications
 {
+#if NET6_0_OR_GREATER
+    // Add the same signature for the OptionSet.Add method as found in the NDesk.Options package for net48
+    internal static class OptionsExtensions
```

Comment:
maybe a better solution is to switch to https://github.com/DynamoDS/Dynamo/blob/83161e3be55c43aaf0ae0342cae80c66a0edbcbe/src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj#L14

like we use in newer parts of the repo.

Reply:
Is it strictly necessary to keep combability here? I think we can get close enough. Maybe something we do for 3.0.

Reply:
Compatibility was not the main reason.
The main reason was to minimize the `#if def` code for dotnet6 and maximize common code